improver/calibration/reliability_calibration.py

```diff
@@ -488,6 +488,289 @@ def process(self, cubes, coordinates=None):
         return result
 
 
+class ManipulateReliabilityTable(PostProcessingPlugin):
```

Comment:
Importing the PostProcessingPlugin does not play nicely when returning a cubelist it would seem; it tries to set a post-processed title, which it cannot do for a cubelist. I suggest you use the BasePlugin instead.

Reply:
done